packages/cli/test/helpers/prepare.js

```diff
@@ -440,6 +440,23 @@ module.exports = async function prepare(session, binaryPath, tmpFixturesDir) {
         },
       }),
     },
+    'vercel-json-configuration-overrides': {
+      'vercel.json': '{}',
+      'package.json': '{}',
+    },
+    'vercel-json-configuration-overrides-merging-prompts': {
+      'vercel.json': JSON.stringify({
+        buildCommand: 'mkdir -p output && echo "1" > output/index.txt',
+      }),
+      'package.json': '{}',
+    },
+    'vercel-json-configuration-overrides-link': {
+      'vercel.json': JSON.stringify({
+        buildCommand: 'mkdir public && echo "1" > public/index.txt',
+      }),
+      '.vercel/output/config.json': JSON.stringify({ version: 3 }),
+      '.vercel/output/public/index.txt': '2',
```

Comment:
Why does this test fixture have `.vercel/output`? Does that change the behavior of `vc link`? I thought that would only affect `vc build`.

Reply:
Yeah I think these can be removed now that the integration test only runs `vc link` instead of also making a deployment